zipline/testing/fixtures.py

```diff
@@ -480,8 +548,10 @@ class WithBcolzDailyBarReader(WithTmpDir, WithSimParams):
     """
     BCOLZ_DAILY_BAR_PATH = 'daily_equity_pricing.bcolz'
     BCOLZ_DAILY_BAR_LOOKBACK_DAYS = 0
-    BCOLZ_DAILY_BAR_FROM_CSVS = False
     BCOLZ_DAILY_BAR_USE_FULL_CALENDAR = False
+    BCOLZ_DAILY_BAR_START_DATE = alias('START_DATE')
+    BCOLZ_DAILY_BAR_END_DATE = alias('END_DATE')
+    _write_method_name = 'write'
```

Comment:
Might be worth commenting that this exists to be overridden in a subclass, and that tests that want to change the write method should inherit from the other mixin rather than overridding this attribute (which is why the name is lowercase).